docs/roadmap.md

```diff
@@ -6,39 +6,40 @@ For a general overview, see the [readme](https://github.com/microsoft/ProjectReu
 
 ## 2021 Roadmap
 
-Here's how we intend to roll out the Windows App SDK. To subscribe to updates, subscribe to [this discussion](https://github.com/microsoft/ProjectReunion/discussions/507). *(all times/features/releases are tentative and subject to change)*: 
+Here's how we intend to roll out the Windows App SDK. To subscribe to updates, subscribe to [this discussion](https://github.com/microsoft/ProjectReunion/discussions/507). *(all times/features/releases are tentative and subject to change)*:
 
 ### Packaged Desktop Apps
 As a [packaged MSIX desktop app](https://docs.microsoft.com/windows/msix/overview) (WPF, WinForms, Win32, console), here's what and when you (tentatively) can expect from the Windows App SDK!
 
-🌟 |      Windows App SDK 0.5      |         Windows App SDK 0.8    |       Windows App SDK 1.0 (Q4 2021)     | 
+🌟 |      Windows App SDK 0.5      |         Windows App SDK 0.8    |       Windows App SDK 1.0 (Q4 2021)     |
 -- | ------------------------------- | --------------------------------- | ------------------------------- |
-[WinUI 3](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md#winui-3) | ✅ Supported  | ✅ Supported | ✅ Supported | 
-[Text rendering](https://docs.microsoft.com/en-us/windows/apps/project-reunion/dwritecore) | ✅ Supported | ✅ Supported | ✅ Supported | 
-[Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview) | ✅ Supported | ✅ Supported | ✅ Supported | 
-[App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111) | ❌ Not Supported | 🔄 Experimental | ✅ Supported | 
-[Power state notifications](https://github.com/microsoft/ProjectReunion/issues/111) | ❌ Not Supported | 🔄 Experimental | ✅ Supported | 
+[WinUI 3](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md#winui-3) | ✅ Supported  | ✅ Supported | ✅ Supported |
+[Text rendering](https://docs.microsoft.com/en-us/windows/apps/project-reunion/dwritecore) | ✅ Supported | ✅ Supported | ✅ Supported |
+[Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview) | ✅ Supported | ✅ Supported | ✅ Supported |
+[App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111) | ❌ Not Supported | 🔄 Experimental | ✅ Supported |
+[Power state notifications](https://github.com/microsoft/ProjectReunion/issues/111) | ❌ Not Supported | 🔄 Experimental | ✅ Supported |
 Windowing | ❌ Not Supported | ❌ Not Supported | ✅ Supported |
 [Push Notifications](https://github.com/microsoft/ProjectReunion/blob/main/specs/PushNotifications/PushNotifications-spec.md) | ❌ Not Supported | ❌ Not Supported | 🔄 Experimental |
```

Comment:
Not your text, but with this can we update Push Notifications to use sentence casing?

```suggestion
[Push notifications](https://github.com/microsoft/ProjectReunion/blob/main/specs/PushNotifications/PushNotifications-spec.md) | ❌ Not Supported | ❌ Not Supported | 🔄 Experimental |
```